packages/neon/neon/lib/src/models/app_implementation.dart

```diff
@@ -19,18 +19,39 @@ import 'package:provider/provider.dart';
 import 'package:rxdart/rxdart.dart';
 import 'package:vector_graphics/vector_graphics.dart';
 
+/// Base implementation of a Neon app.
+///
+/// It is mandatory to provide a precompiled svg under `assets/app.svg.vec`.
+/// SVGs can be precompiled with `https://pub.dev/packages/vector_graphics_compiler`
```

Comment:
This is duplicated below, is that intended?

Reply:
Yes.
This information is important enough that it should be mentioned in the documentaion of the class.

It should obviously also be at the relevant field.